ems-business-logic/src/main/java/com/devtechgroup/ems/business/logic/service/impl/EmployeeService.java

```diff
@@ -0,0 +1,100 @@
+package com.devtechgroup.ems.business.logic.service.impl;
+
+import com.devtechgroup.ems.business.logic.adapter.EmployeeAdapter;
+import com.devtechgroup.ems.business.logic.model.EmployeeDto;
+import com.devtechgroup.ems.business.logic.service.IEmployeeService;
+import com.devtechgroup.ems.data.access.entity.Authority;
+import com.devtechgroup.ems.data.access.entity.Employee;
+import com.devtechgroup.ems.data.access.repository.IAuthorityRepository;
+import com.devtechgroup.ems.data.access.repository.IEmployeeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Service
+@Transactional
+public class EmployeeService implements IEmployeeService {
+
+    private IEmployeeRepository _employeeRepository;
+
+    private IAuthorityRepository _authorityRepository;
+
+    private EmployeeAdapter employeeAdapter;
+
+    @Autowired
+    public EmployeeService(IEmployeeRepository employeeRepository, IAuthorityRepository authorityRepository){
+        _employeeRepository = employeeRepository;
+        _authorityRepository = authorityRepository;
+    }
+
+    @Override
+    public EmployeeDto createEmployee(EmployeeDto employeeDto) {
+
+        Employee newEmployee = EmployeeAdapter.adapt(employeeDto);
```

Comment:
Regarding previous note where you had EmployeeAdapter as private field. See how here you are using static class as it should. SO just remove the private field.